### PR TITLE
Update workstation scripts to be more broadly useful

### PR DESCRIPTION
**Pull request by Gerg**
Follow-up to https://github.com/cloudfoundry/capi-ci/pull/52

### CODE REVIEW

**Review by moleske — COMMENTED** [review]
I haven't tested every command in this pr, but it seems reasonable to me.  Are you waiting for something in particular to change this to not a draft and get back to the main branch?

**Comment by Gerg** [comment]
> I haven't tested every command in this pr, but it seems reasonable to me. Are you waiting for something in particular to change this to not a draft and get back to the main branch?

I had a secret list of things I wanted to do, which I have now mostly accomplished. I converted to a full PR.

File: README.md [hunk 1]
```diff
@@ -160,9 +92,239 @@ testing
...
     capi-workspaces     		 List and connect to existing K8s CAPI workspaces
     create-capi-workspace 		 Create a CAPI workspace in K8s cluster
 ```
+
+## Using the Bosh Lite Pool
+
+As a perk, [CAPI
+approvers](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-interfaces.md)
+get access to a pool of cf-deployment environments, deployed on bosh lites in a
+CFF-managed IaaS account. These environments are displosable, and are only
```
Comment (evanfarrar) [inline README.md:108]: s/displosable/disposable/

**Review by evanfarrar — APPROVED** [review]
Found a typo in the readme, but otherwise LGTM

**Review by tcdowney — DISMISSED** [review]
thx

**Review by moleske — APPROVED** [review]
whoopsie doodle, didn't realize making a commit to fix the spelling error would dismiss a perfectly [cromulent](https://frinkiac.com/meme/S07E16/104037.jpg?b64lines=ICJFbWJpZ2dlbnM_IiBIbSwgSSBuZXZlcgogaGVhcmQgdGhhdCB3b3JkIGJlZm9yZSBJCiBtb3ZlZCB0byBTcHJpbmdmaWVsZC4gSQogZG9uJ3Qga25vdyB3aHkuIEl0J3MgYQogcGVyZmVjdGx5IGNyb211bGVudCB3b3JkLg==) review.  apologies tim

**Review by tcdowney — APPROVED** [review]
Looks even better to me!

**Pull request merged**